ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -246,6 +246,7 @@ The\ group\ must\ be\ an\ integer = The group must be an integer
 The\ offset\ unit\ must\ be\ an\ integer = The offset unit must be an integer
 The\ unit\ must\ be\ an\ integer = The unit must be an integer
 This\ message\ is\ part\ of\ group = This message is part of group
+This\ contact\ does\ not\ exist\ anymore,\ you\ can\'t\ send\ them\ a\ message = This contact does not exist anymore, you can\'t send them a message.
```

Comment:
We've moved away from using spaces in the translation templates, and we use a dot separator instead. 
https://docs.communityhealthtoolkit.org/core/overview/translations/#translating-static-text

> Otherwise create a new key and default English value. Keys must be all lower case, dot separated, and descriptive but not verbose. The values should include as much text as possible (eg: trailing punctuation), and must not contain any markup.